man/docker-login.1.md

```diff
@@ -18,9 +18,11 @@ Register or log in to a Docker Registry located on the specified
 do not specify a `SERVER`, the command uses Docker's public registry located at
 `https://registry-1.docker.io/` by default.  To get a username/password for Docker's public registry, create an account on Docker Hub.
 
+`docker login` requires you to use `sudo` or be `root`.
```

Comment:
This assumption of being root is following in the theme of a limited user _not_ being in the host group of `docker`. 
Though, if the user decides against that security best-practice, and adds their user to the `docker` group, then sudo or root is not strictly required.